src/test/ui/const-generics/issues/issue-62220.rs

```diff
@@ -0,0 +1,22 @@
+// check-pass
+#![allow(incomplete_features)]
+
+#![feature(const_generics)]
+pub struct Vector<T, const N: usize>([T; N]);
+
+pub type TruncatedVector<T, const N: usize> = Vector<T, { N - 1 }>;
```

Comment:
@eddyb @varkor Shouldn't `error: constant expression depends on a generic parameter` happen here or is this cause we're not applying WF for type aliases... but then why won't it happen for the usage below?

Reply:
We're not doing const WF at all except for array types, today.
#70107 aims to fix that.

Reply:
Ah; forgot that hadn't landed yet. Let's merge this PR like this then once the other comment has been addressed and we can make adjustments later in the other.